src/edu/stanford/nlp/sempre/Main.java

```diff
@@ -11,6 +11,7 @@
 public class Main implements Runnable {
   @Option public boolean interactive = false;
   @Option public boolean server = false;
+  @Option public boolean interactiveLearning = false;
```

Comment:
Put gloss on these flags...having `interactive` and `interactiveLearning` is going to be very confusing.